mrp_project_wo_timesheet/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# © 2025 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See https://www.gnu.org/licenses/lgpl-3.0.html
+{
+    "name": "MRP Project WO Timesheet",
+    "summary": """
+        Add an extension between workorders to generate timesheets based on manufacturing productivity.
+    """,
+    "author": "Solvos",
+    "license": "LGPL-3",
+    "version": "15.0.1.0.0",
+    "category": "Manufacturing",
+    "website": "https://github.com/solvosci/slv-manufacture",
+    "depends": ["mrp_project"],
```

Comment:
Missing dependency, as we're using timesheet capabilities (like e.g. `project_id` field) for account analytic lines:
```suggestion
    "depends": ["hr_timesheet", "mrp_project"],
```